bin/kn

```diff
@@ -114,6 +116,82 @@ function validate_pwd() {
   fi
 }
 
+function resolve_links() {
+  # parameters
+  source="$1"
+  # resolve $source until the file is no longer a symlink
+  while [ -h "$source" ]; do
+    dir="$(cd -P "$(dirname "$source")" && pwd)"
+    # if $dir was a relative symlink, we need to resolve it relative to
+    # the path where the symlink file was located
+    source="$(readlink "$source")"
+    [[ $dir != /* ]] && dir="$dir/$dir"
+  done
+  echo "$source"
+}
+
+function upgrade() {
+  # version is first arg or otherwise default
+  version=${1:-$kubenow_version}
+  echo "upgrade version: $version"
+  branch=""
+  case "$version" in
+  latest)
+    branch="$(curl https://api.github.com/repos/kubenow/KubeNow/releases |
+      jq -r '.[0].tag_name')"
+    ;;
+  latest-stable)
+    echo "lateststable"
```

Comment:
Hey @andersla Was this meant to be a debug echo, or is it meant to be there for a reason? Tack!

Reply:
Probably a debug echo - will remove:) Thanks for spotting